usaspending_api/awards/models_matviews.py

```diff
@@ -7,73 +7,55 @@
 
 
 class UniversalTransactionView(models.Model):
-    transaction_id = models.IntegerField(blank=False, null=False, primary_key=True)
+    keyword_string = models.TextField()
+    award_id_string = models.TextField()
     action_date = models.DateField(blank=True, null=False)
     fiscal_year = models.IntegerField()
     type = models.TextField(blank=True, null=True)
     action_type = models.TextField()
     award_id = models.IntegerField()
     award_category = models.TextField()
     total_obligation = models.DecimalField(
-        max_digits=15, decimal_places=2, blank=True,
-        null=True)
+        max_digits=15, decimal_places=2, blank=True, null=True)
     total_obl_bin = models.TextField()
     fain = models.TextField()
     uri = models.TextField()
     piid = models.TextField()
     federal_action_obligation = models.DecimalField(
-        max_digits=20, db_index=True, decimal_places=2, blank=True,
```

Comment:
Don't see the corresponding `federal_action_obligation` index being removed in the json?

Reply:
Django isn't managing this "model" so no need for it listed in code